README.md

```diff
@@ -369,7 +370,120 @@ lc-serve deploy jcloud --app endpoints:app
 
 </details>
 
----
+## 🗝️ Using Secrets during Deployment
+
+You can use secrets during app deployment by passing a secrets file to deployment with the `--secrets` flag. The secrets file should be a `.env` file containing the secrets.
+
+```bash
+lcserve deploy jcloud app --secrets .env
+```
+
+<details>
+<summary>Show details</summary>
+
+Let's take an example of a simple FastAPI app that uses an external Redis instance for rate-limiting endpoints.
+
+### 🚦 Deploy a FastAPI app with redis based rate-limiting
```

Comment:
Can we please change this to an example that uses `OPENAI_API_KEY` instead of redis?

Reply:
sure

Reply:
ok revised